### PR TITLE
renamed MS.VS.CodeAnalysis.CodeLens to MS.VS.LanguageServices.CodeLens

### PR DESCRIPTION
**Pull request by heejaechang**
fix https://github.com/dotnet/roslyn/issues/35111

### CODE REVIEW

**Comment by heejaechang** [comment]
tagging @jinujoseph @tmat

File: Roslyn.sln [hunk 1]
```diff
@@ -394,7 +394,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.Exte
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.CodeAnalysis.ExternalAccess.TypeScript", "src\Tools\ExternalAccess\TypeScript\Microsoft.CodeAnalysis.ExternalAccess.TypeScript.csproj", "{1638FB04-0298-4341-B5E0-8A13B4823C81}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.CodeAnalysis.CodeLens", "src\VisualStudio\CodeLens\Microsoft.VisualStudio.CodeAnalysis.CodeLens.csproj", "{5E6E9184-DEC5-4EC5-B0A4-77CFDC8CDEBE}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.VisualStudio.LanguageServices.CodeLens", "src\VisualStudio\CodeLens\Microsoft.VisualStudio.LanguageServices.CodeLens.csproj", "{5E6E9184-DEC5-4EC5-B0A4-77CFDC8CDEBE}"
```
Comment (jasonmalinowski) [inline Roslyn.sln]: Generally curious: why is this a separate project in the first place?

Comment (tmat) [reply Roslyn.sln]: Good question.

Comment (heejaechang) [reply Roslyn.sln]: it loads to code lens OOP process. putting this in existing MS.VS.LanguageServices.dll caused bunch of dependencies we don't want to be loaded in the process. and bunch of MEF issues to be logged.

**Pull request merged**